bluepyefe/recording.py

```diff
@@ -187,10 +189,20 @@ def standardize_trace(self, config_data, reader_data):
             )
 
         # Convert current to nA
+        amp = None
+        hypamp = None
         if "i_unit" in config_data and config_data["i_unit"] is not None:
             current = to_nA(reader_data["current"], config_data["i_unit"])
+            if "amp" in reader_data:
+                amp = to_nA(reader_data["amp"], config_data["i_unit"])
+            if "hypamp" in reader_data:
+                hypamp = to_nA(reader_data["hypamp"], config_data["i_unit"])
         elif "i_unit" in reader_data and reader_data["i_unit"] is not None:
             current = to_nA(reader_data["current"], reader_data["i_unit"])
+            if "amp" in reader_data:
+                amp = to_nA(reader_data["amp"], reader_data["i_unit"])
+            if "hypamp" in reader_data:
+                hypamp = to_nA(reader_data["hypamp"], reader_data["i_unit"])
```

Comment:
I think there is also the possibility of having amp and hypamp in config_data. I would suggest checking for i_unit in config and amp in config, i_unit in config and amp in reader, i_unit in reader and amp in config, and i_unit in reader and amp in reader. So that we are sure of not missing any case.